gpkitmodels/GP/aircraft/wing/boxspar.py

```diff
@@ -1,48 +1,75 @@
-" cap spar "
-from gpkit import Model, Variable, Vectorize
-from sparloading import SparLoading
-from gustloading import GustL
+" box spar "
+from gpkit import Model, parse_variables
+from .sparloading import SparLoading
+from .gustloading import GustL
+
+#pylint: disable=exec-used, undefined-variable, unused-argument, invalid-name
 
 class BoxSpar(Model):
-    "cap spar model"
+    """ Box Spar Model
+
+    Scalar Variables
+    ----------------
+    rhocfrp         1.6     [g/cm^3]    density of CFRP
+    E               2e7     [psi]       Young modulus of CFRP
+    W                       [lbf]       spar weight
+    wlim            0.15    [-]         spar width to chord ratio
+    tshearmin       0.012   [in]        min gauge shear web thickness
+    g               9.81    [m/s^2]     gravitational acceleration
+    mfac            0.97    [-]         curvature knockdown factor
+    rhocore         0.036   [g/cm^3]    foam core density
+    tcoret          0.02    [-]         core to thickness ratio
+
+    Variables of length N-1
+    -----------------------
+    hin                     [in]        height between caps
+    I                       [m^4]       spar x moment of inertia
+    Sy                      [m^3]       section modulus
+    dm                      [kg]        segment spar mass
+    w                       [in]        spar width
+    t                       [in]        spar cap thickness
+    tshear                  [in]        shear web thickness
+    tcore                   [in]        core thickness
+
+    Upper Unbounded
+    ---------------
+    W
+
+    Lower Unbounded
+    ---------------
+    Sy
+
+    LaTex Strings
+    -------------
+    rhocfrp                 \\rho_{\\mathrm{CFRP}}
+    wlim                    w_{\\mathrm{lim}}
+    tshearmin               t_{\\mathrm{shear-min}}
+    mfac                    m_{\\mathrm{fac}}
+    rhocore                 \\rho_{\\mathrm{core}}
+    hin                     h_{\\mathrm{in}_i}
+    I                       I_i
+    Sy                      S_{y_i}
+    dm                      \\Delta{m}
+    w                       w_i
+    t                       t_i
```

Comment:
Why do these have `_i`? Just because they're vectors? But they could become matrices if you do fleet design...

Reply:
Well I want to treat the latex representation as how I would write it in a paper and I've written it as `w_i` and `t_i` in the papers I've done so far. But open to discussion!

Reply:
Sounds good to me!